azure/ConsiderationsForServiceDesign.md

```diff
@@ -84,6 +84,74 @@ As you build out your service and API, there are a number of decisions that can
 
 :ballot_box_with_check: **YOU SHOULD** implement [conditional requests](https://tools.ietf.org/html/rfc7232) early. This allows you to support concurrency, which tends to be a concern later on.
 
+## Use Good Names
+
+Good names for resources, properties, operations, and parameters are essential to a great developer experience.
+
+Resources are described by nouns. Resource and property names must be descriptive and easy for customers to understand.
```

Comment:
I think it would be good to call out that endpoints should be described by verbs. I've had at least once instance where a service team used a noun as an endpoint name and it would be good to point them to these guidelines.

Reply:
OperationIds are not part of the formal API so they are covered in the [OpenAPI Style Guide](https://github.com/Azure/azure-api-style-guide/blob/main/openapi-style-guide.md#operationid).

For Post actions where the action is the last segment of the path, this is covered in the [API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#user-content-actions-action-name-is-verb).